data-finding/src/DataReader.py

```diff
@@ -4,176 +4,121 @@
 import json
 import csv
 import os
-import datetime
-
-class FileReader:
-
-  initPath = ""
-  data_mapper = json.load(open(os.getcwd() + "/data-finding/assets/mapper.json"))
-  formatter = []
-
-  def setInitPath(self, path):
+import logging
+
+#CleanLine cleans the data to remove certain data points from the CSV based on the format provided by createFormatter
+def cleanLine(line):
+  output = []
+  for i in range(len(formatter)):
+    #Must always check if location name is found
+    #Must also always check if there are a total len of formmater in each line
+    if data_mapper[str(i+1)]["type"] == "date":
```

Comment:
if you're gonna access the same thing @15, 19 you should store it as a var so you don't need to get it twice/readability